register.js

```diff
@@ -0,0 +1,54 @@
+
+
+document.getElementById('submitButton').addEventListener('click', (e) => {
+    //Prevents default form action from happening
+    e.preventDefault();
+
+    //Get Previous Items as array
+    let previousClients = getLocalStorage()
+    
+    console.log(previousClients, "Previous Clients")
+
+    //Get Input Values
+    let nameValue = document.querySelector('input[type=text]').value;
+    let emailValue = document.querySelector('input[type=mail]').value;
+    let phoneValue = document.querySelector('input[type=tel]').value;
+    validateForm(nameValue,emailValue,phoneValue)
```

Comment:
When do you call this Function ?

Reply:
i should call the validation before adding the JSON object to the local storage, after making sure that all of the above is ok i can add the information to the local storage.

Reply:
Can you try to call after focusing out of the input ? I mean if the email is wrong it would be better if you tell taht this wrong once he goes out from the input. And yeah, do the double check before adding the JSON object